### ISSUE TITLE
$cookies not working in 1.3.15

### ISSUE THREAD

**Issue by mshaaban0** [angular/angular.js#11554]
I have a problem with ``` $cookies ``` it's throwing an undefined function when using ```$cookies.get()``` or ``` $cookies.put() ``` i was using ``` $cookieStore ``` but the problem is i have angular running in a wordpress site custom page and when i set a cookie for user login while he in the sub directory page the cookies path become /book for example , i tried to use $cookies to define the path but it threw an undefined function error.

Example of a code
PS: non of get() , getAll or getObject works as well.
```
maticApp.service('userData', ['$cookieStore', '$cookies', function ($cookieStore, $cookies) {
    
    return {
        get: function () {
            if (isEmpty(userData)) {
                userData = this.getCookie();
            }
            return userData;
        },
        set: function (data) {
            userData = data;
        },
        setCookie: function (data) {
            var options = {
                path: '/'
            }
            $cookie.put('userData', data, options);
        },
        getCookie: function () {
            var user = $cookie.get('userData');
            if (typeof(user) == 'undefined'){
                user={
                    phone : null,
                    VerificationCode:null,
                    device_id:null,
                    error:null,
                    firstName:null,
                    lastName:null,
                    email:null
                }
            }
            return user;
        },
        removeCookie: function () {
            $cookieStore.remove('userData');
        }
    };
}]);
```

**Comment by tskaggs** [comment]
+1 Same comment... $cookies not working.

**Comment by shahata** [comment]
`$cookies.get` and `$cookies.put` are methods available only in Angular 1.4
You should refer to 1.3 docs to learn about how to use `$cookies` in this version.
